### ISSUE TITLE
`:cloverage` plugin doesn't show coverage for cljs files

### ISSUE THREAD

**Issue by Valdermeyder** [lambdaisland/kaocha-cljs2#7]
When tests are run using `:kaocha.type/cljs2` the coverage is shown only for "*.cljc" files but not for the "*.cljs".
Is it a known issue?

**Comment by plexus** [comment]
Yes, Cloverage is a Clojure-only project. You would have to write coverage instrumentation for ClojureScript from scratch, it's a completely different ball game. I'm not aware of any tools yet that can do it.

There's a cloverage issue but understanably with not much content: https://github.com/cloverage/cloverage/issues/182

Some discussion also here: https://github.com/thheller/shadow-cljs/issues/562
